src/Components/Orders/Orders.component.tsx

```diff
@@ -0,0 +1,49 @@
+import { useState } from "react";
+import Header from "../Header";
+import SideNavBar from "../SideNavBar";
+import ActiveOrders from "./ActiveOrders";
+import VoidedOrders from "./VoidedOrders";
+
+function Orders() {
+  const [openTab, setOpenTab] = useState(1);
```

Comment:
Think of a way to handle this dynamically.
Scenario one; If you have 10 tabs or even 5, will you have 10/5 if blocks?